python/cuml/fil/fil.pyx

```diff
@@ -521,7 +509,9 @@ class ForestInference(Base,
                                  algo='auto',
                                  threshold=0.5,
                                  storage_type='auto',
-                                 blocks_per_sm=0):
+                                 blocks_per_sm=0,
```

Comment:
Optional: if `**kwargs` is also used in methods of `ForestInference`, there would be no need to thread through each new parameter.